docs/reference/method/MongoDBDatabase-command.txt

```diff
@@ -53,7 +53,7 @@ result document:
 
    $database = (new MongoDB\Client)->test;
 
-   $cursor = $database->command(['isMaster' => 1]);
+   $cursor = $database->command(['hello' => 1]);
```

Comment:
I wonder if it'd be preferable to use `ping` here, since it's portable across all server versions and also returns just a single document.

Reply:
Changed to use ping.